src/vm/memory/relocatable.zig

```diff
@@ -222,11 +233,17 @@ pub const MaybeRelocatable = union(enum) {
     /// Whether the MaybeRelocatable is a relocatable or not.
     /// # Returns
     /// true if the MaybeRelocatable is a relocatable, false otherwise.
-    pub fn isRelocatable(self: Self) bool {
-        return switch (self) {
-            .relocatable => true,
-            .felt => false,
-        };
+    pub fn isRelocatable(self: MaybeRelocatable) bool {
+        return std.meta.activeTag(self) == .relocatable;
+    }
+
+    /// Returns whether the MaybeRelocatable is a felt or not.
+    ///
+    /// # Returns
+    ///
+    /// `true` if the MaybeRelocatable is a felt, `false` otherwise.
+    pub fn isFelt(self: MaybeRelocatable) bool {
```

Comment:
good point - do you mind also replacing `!x.isRelocatable()` with `x.isFelt()` where relevant?

Reply:
sure!